src/features/controls.ts

```diff
@@ -217,6 +227,54 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
         {type: String, attribute: 'field-of-view', hasChanged: () => true})
     fieldOfView: string = DEFAULT_FIELD_OF_VIEW;
 
+    @style({
+      intrinsics: {
+        basis: [
+          numberNode(-Infinity, 'rad'),
+          numberNode(Math.PI / 8, 'rad'),
+          numberNode(0, 'm')
+        ],
+        keywords: {auto: [null, null, null]}
+      },
```

Comment:
Let's pull the intrinsics declarations out of the decorator so that it is a little less busy here (similarly to what we did with the other ones).